README.md

```diff
@@ -1,9 +1,15 @@
 # Arkime AWS All-in-One
 
-The goals of this project are 1) provide a demonstration of how Arkime can be deployed in a cloud-native manner and 2) provide scripting to enable users to easily begin capturing the traffic in their existing AWS cloud infrastructure.
+The goals of this project:
+1) Demonstrate how Arkime can be deployed in a cloud-native manner
+2) Provide scripting to enable users to easily begin capturing the traffic in their existing AWS cloud infrastructure
 
 The AWS Cloud Development Kit (CDK) is used to perform infrastructure specification, setup, management, and teardown.  You can learn more about infrastructure-as-code using the CDK [here](https://docs.aws.amazon.com/cdk/v2/guide/home.html).
 
+## Quick Start Guide
+TODO
```

Comment:
What are the minimal steps to get started?

Reply:
1. Install the prereqs
2. run ./manage_arkime.py cluster-create --name ClusterName (see manage_arkime.py cluster-create --help for important options)
3. run ./manage_arkime.py vpc-add --cluster-name ClusterName --vpc-id VPCID # to add the cluster
4. run ./manage_arkime.py get-login-details --name ClusterName # to see default login details